server/worker/batch_activities.go

```diff
@@ -0,0 +1,50 @@
+package worker
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	kitlog "github.com/go-kit/log"
+)
+
+const batchScriptsName = "batch_scripts"
```

Comment:
When you [add the job](https://github.com/fleetdm/fleet/blob/0ab4280616432abe06cd1692ef55078a2c2a274d/server/datastore/mysql/scripts.go#L1988-L1993) you use [`batch_activity`](https://github.com/fleetdm/fleet/blob/982e138792e8bbf32b59ddfb5b50f26acae3de1b/server/fleet/scripts.go#L645), so this needs to match.

Reply:
Check again and ye will see